src/components/form-draft/publish.vue

```diff
@@ -242,7 +242,7 @@ export default {
       }
     },
     "introduction": [
-      "You are about to make this Draft the published version of your Form. This will finalize any changes you have made to the Form definition and its attached Media Files.",
+      "You are about to make this Draft the published version of your Form. This will finalize any changes you have made to the Form definition and Form Attachments.",
```

Comment:
Hmm maybe I should have kept "its": the text could be "its Form Attachments" or maybe even just "its Attachments".

Reply:
I think it's ok as-is. It's not clear whether "its" refrences Form or Form definition anyway.